README.mdx

```diff
@@ -5,13 +5,42 @@
 
 # Linea documentation
 
+## Quick Links 🔗
```

Comment:
As mentioned in the contribution guidelines below (line 73), we use the Consensys style guidelines, which means sentence case for all headings and subheadings. Could you implement this throughout please?

I may push up some changes to extract this and some other main points from that site to add clarity. 

There is also unnecessary capitalisation in the bulleted list which needs adjustment